tests/chan_select.rs

```diff
@@ -62,13 +82,8 @@ fn chan_select_simple() {
     recv_assert_eq_close!("Hello, World!".to_owned(), rcs.recv());
 }
 
-#[cfg(feature = "chan_select")]
 #[test]
 fn chan_select_add_ret() {
```

Comment:
I'm kinda thinking this test could just be removed - the whole idea with the `_ret` stuff was to allow users to select the types returned from `sel.wait()` (in this test the `ChanToRead`). But that whole thing is gone now, so I'm not sure this test adds anything.